packages/ckeditor5-engine/src/conversion/downcastdispatcher.js

```diff
@@ -636,6 +837,40 @@ function shouldMarkerChangeBeConverted( modelPosition, marker, mapper ) {
 	return !hasCustomHandling;
 }
 
+function getEventName( type, data ) {
+	const name = data.item.name || '$text';
+
+	return `${ type }:${ name }`;
+}
+
+function getElementFromChange( entry ) {
+	const { range, position, type } = entry;
+
+	return type === 'attribute' ? range.start.nodeAfter : position.parent;
+}
+
+function rangeIteratorValueToEventData( value ) {
```

Comment:
This function name is a bit confusing, maybe `walkerValueToEventData`, or `treeWalkerValueToEventData` would be better?